core/dbt/clients/agate_helper.py

```diff
@@ -135,12 +135,12 @@ def as_matrix(table):
     return [r.values() for r in table.rows.values()]
 
 
-def from_csv(abspath, text_columns):
+def from_csv(abspath, text_columns, delimiter = ","):
```

Comment:
**Question:** Should we add support for more than just `delimiter` here? Any/all `**kwargs` supported by `agate.from_csv` / Python's stdlib CSV reader?

**Thoughts:**
- That can be out of scope for now / for this PR
- It would risk locking us further into using `agate` (which I don't love), or at least the Python stdlib `csv` reader (which I don't mind as much)